src/contexts/Articles.jsx

```diff
@@ -0,0 +1,13 @@
+import { createContext, useState } from 'react';
+
+export const ArticlesContext = createContext();
+
+export const ArticlesProvider = ({ children }) => {
+    const [articles, setArticles] = useState();
+
+    return (
+        <ArticlesContext.Provider value={{articles, setArticles}}>
+            {children}
+        </ArticlesContext.Provider>
+    )
```

Comment:
You've implemented context well here - though I'm not sure the articles is a great use case for using context as it's only currently used in one place in the app. No need to change though